test/runtests.jl

```diff
@@ -50,13 +50,28 @@ end
 
   @testset "Shortcut C" begin
     m = Metalhead.resnet(Metalhead.basicblock, :C;
-                         channel_config = [1, 1],
-                         block_config = [2, 2, 2, 2])
+      channel_config = [1, 1],
+      block_config = [2, 2, 2, 2])
```

Comment:
Try not to change the other code.

Reply:
Whoops, must've snuck in there accidentally, apologies

Reply:
No problem!